src/golang/cmd/server/handler/v2/integrations_workflows_get.go

```diff
@@ -0,0 +1,136 @@
+package v2
+
+import (
+	"context"
+	"fmt"
+	"github.com/aqueducthq/aqueduct/cmd/server/handler"
+	aq_context "github.com/aqueducthq/aqueduct/lib/context"
+	"github.com/aqueducthq/aqueduct/lib/database"
+	"github.com/aqueducthq/aqueduct/lib/dynamic"
+	"github.com/aqueducthq/aqueduct/lib/errors"
+	"github.com/aqueducthq/aqueduct/lib/models"
+	"github.com/aqueducthq/aqueduct/lib/models/shared"
+	"github.com/aqueducthq/aqueduct/lib/repos"
+	"github.com/aqueducthq/aqueduct/lib/workflow/operator"
+	"github.com/google/uuid"
+	"net/http"
+)
+
+// This file should map directly to
+// src/ui/common/src/handlers/v2/IntegrationsWorkflows.ts
```

Comment:
I have the impression that UI file name should include http method (`Get`) as well